### ISSUE TITLE
Growl Notification can not handle RTL Correctly, Inversed Images

### ISSUE THREAD

**Issue by AlBannaTechno** [HandyOrg/HandyControl#799]
**Hi**,
I work on an RTL app, but I just noticed that the growl notification have an issue, might be two,
see,

![2021-04-21_062015_2](https://user-images.githubusercontent.com/13814190/115675105-948d5900-a34e-11eb-8c0a-dcec92946d21.png)

### The issue on the left side, Local Growl

```xaml
     <hc:ScrollViewer
                Grid.Column="1"
                Grid.Row="0"
                Background="Transparent"
                VerticalScrollBarVisibility="Hidden" HorizontalAlignment="Right" IsInertiaEnabled="True"
                IsPenetrating="True">
                <StackPanel
                    hc:Growl.Token="AllMToken"
                    hc:Growl.GrowlParent="True" VerticalAlignment="Top" Margin="0,10,10,10" />
            </hc:ScrollViewer>
```

the alignment works correctly, but the image of states was inversed: this means that this image inherited the FlowDirection from its parent
	which should not happen, it should explicitly be `FlowDirection="LeftToRight"`

### The issue on the right side : Global Growl
```C#
Growl.ErrorGlobal() // and so on.
```
this seems to use LTR direction, so it produces a culture-specific alignment issue, `view RTL language in LTR context`
so the question here is, how to change the global Growl  **FlowDirection**, if it is not possible, then it is an issue 😄

**Comment by ghost1372** [comment]
@AlBannaTechno  Thank you, I will send a PR to solve this problem soon

**Comment by ghost1372** [comment]
if #801  accepted, you can use this code:

```
Growl.AskGlobal(new GrowlInfo
            {
                Message = "سلام خوش آمدید",
                FlowDirection = FlowDirection.RightToLeft,
                ActionBeforeClose = b =>
                {
                    if (!b) return true;
                    return true;
                }
            });
```

**Comment by NaBian** [comment]
closed in 2fbcc1a6
